mlflow/pipelines/utils/metrics.py

```diff
@@ -87,6 +93,8 @@ def _get_builtin_metrics(tmpl: str) -> str:
     """
     if tmpl == "regression/v1":
         return BUILTIN_REGRESSION_PIPELINE_METRICS
+    if tmpl == "classification/v1":
```

Comment:
`else if`?

Reply:
line 95 returns - would you still like me to change this?

Reply:
I prefer if-else but either way it works. No strong opinion.